yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,139 @@
+description: >
+    Implement to manage VPD in the system.
+
+methods:
+    - name: WriteKeyword
+      description: >
+          Method to update a single keyword's value based on the input
+          parameters.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Path where the data resides(D-Bus inventory path/EEPROM path
+                etc).
+          - name: writeData
```

Comment:
name: paramsToWriteData
description: > Parameters in required format to write the keyword data.